CondCore/EcalPlugins/plugins/getPayloadData.py

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/env python
```

Comment:
@ggovi, thanks - I suppose then the instance in `CondCore/EcalPlugins/plugins/` proposed here, becomes superfluous.